app/views/topics/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for(@topic) do |f| %>
```

Comment:
Aim to have partial templates like `_form.html.erb` only use local variables, not `@instance_variables`. This makes them more reusable by being less dependent on the variable names chosen in controller actions.

Refactor this partial, and any other templates that use it (e.g. line 2 in `app/views/topics/edit.html.erb`), to use local variables via the `locals` option.